doc/ecmp/inner_packet_hashing_test_plan.md

```diff
@@ -0,0 +1,40 @@
+# Test Plan for inner packet hashing in ECMP
+###### Revision
+| Rev |     Date    |       Author       | Change Description                |
+|:---:|:-----------:|:------------------:|-----------------------------------|
+| 0.1 | 03/12/2021  |    Anish Narsian   | Initial version                   |
+
+## Overview
+The purpose of this test plan is to describe inner packet hashing tests for ECMP nexthops. The test methodology will be similar to the current hash_test in SONiC. The test sends vxlan encapsulated packets and validates hashing is performed per the inner packet's 5 tuple(src ip, dst ip, src port, dst port, ip proto).
+
+## Test information
+
+### Test configuration
+Currently this test expects that the inner packet hashing is pre-configured on the DUT. In the future, once dynamic Policy Based Hashing feature is implemented, dynamic configuration can be done on the DUT and packet hashing verified per the config.
+
+Beyond hash configuration, base DUT route table configurations are sufficient to perform the test.
+
+### Supported topology
+The test will be supported on the T0 toplogy, it may be enhanced in the future for other topologies.
+
+### High level test details
+1. Send packets to a destination prefix which is pointing to multiple ecmp nexthops. For the T0 topology test we will send it to a dest prefix which is pointing to the T1s.
+3. Vary some tuple of the packet so that the packets hash to different nexthops. The total packets sent in a test is calcualted as follows: 1000 packets sent per ECMP next hop. This translates to 4000 packets in a T0 topology with 4 T1s(ECMP nexthops). All 4000 packets will have varied tuples to get a good distribution of packets to ports.
```

Comment:
@anish-n T0 has 4 LAGs with a single member per LAG and VLAN with 24 members. Does it mean that we are going to send 1000 packets per LAG and expect packets will be distributed across VLAN members?

Reply:
No the direction of packets is the opposite, ~4000 packets sent from the VLAN members and received at the T1, we will check the distribution at the T1